Sources/PointFree/SiteMiddleware.swift

```diff
@@ -201,24 +210,9 @@ private let allowedInsecureHosts: [String] = [
 
 private func isProtected(route: Route) -> Bool {
   switch route {
-  case .about,
-       .admin,
-       .account,
-       .appleDeveloperMerchantIdDomainAssociation,
-       .expressUnsubscribe,
-       .expressUnsubscribeReply,
-       .episode,
-       .feed,
-       .gitHubCallback,
-       .invite,
-       .login,
-       .logout,
-       .pricing,
-       .privacy,
-       .home,
-       .subscribe,
-       .team:
-
+  case .webhooks(.stripe):
+    return true
+  default:
```

Comment:
aw you dont want to be explicit with this stuff? I think it's nice that when adding a route we have to make sure we know whether or not we mean it to be open to the public

Reply:
Yeah...I agree in principle but we're not really using the feature anymore. Even here it's layered security that shouldn't matter. Maybe remove for now?

Reply:
Gonna remove in another PR!